src/main/java/uk/gov/hmcts/reform/unspec/launchdarkly/FeatureToggleService.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.unspec.launchdarkly;
+
+import com.launchdarkly.sdk.LDUser;
+import com.launchdarkly.sdk.server.interfaces.LDClientInterface;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+
+@Slf4j
+@Service
+public class FeatureToggleService {
+
+    public static final LDUser UNSPEC_SERVICE_USER = new LDUser.Builder("civil-unspec-service")
+        .anonymous(true)
```

Comment:
Do you think it'd make sense to add some other type of identifier here? We're probably going to have 2 environments (test and prod) and it would be nice to be able to profile flag values depending on environment. We could add different values for aat/staging/demo and use `SERVICE_FQDN` for preview.